test.yml

```diff
@@ -0,0 +1,13 @@
+github_organization: scalanga-devl
+cloud_nine_region: Europe
+cloud_nine_tenant: erwan.bernard@scality.com
+python_config: 'runners_manager.settings.settings_local'
+runner_pool:
+  - flavor: 'm1.small'
+    image: 'CentOS 7 (PVHVM)'
+    quantity:
+      min: 2
+      max: 4
+    tags:
+      - centos7
+      - small
```

Comment:
You can keep a sample.yml file but please remove any value that represent you, or our field as this repository is public